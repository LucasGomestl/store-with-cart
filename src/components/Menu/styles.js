import styled from "styled-components";

export const Menu = styled.nav`
  grid-column: 1/-1;
  grid-row: 2;
  justify-self: center;
  margin: 35px 0px;
`;

export const MenuItems = styled.ul`
  display: flex;
  justify-content: space-between;
  li {
    margin: 0 2.5vw;
  }
`;
