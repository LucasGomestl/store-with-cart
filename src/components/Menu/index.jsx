import React from "react";
import { Menu, MenuItems } from "./styles";
const index = () => {
  return (
    <Menu data-testid="menu">
      <MenuItems>
        <li>
          <a href="/">Item 1</a>
        </li>
        <li>
          <a href="/">Item 2</a>
        </li>
        <li>
          <a href="/">Item 3</a>
        </li>
        <li>
          <a href="/">Item 4</a>
        </li>
      </MenuItems>
    </Menu>
  );
};

export default index;
