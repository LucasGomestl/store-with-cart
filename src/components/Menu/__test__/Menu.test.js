import React from "react";
import { render } from "@testing-library/react";
import Menu from "../index";

describe("Menu tests", () => {
  it("Should render menu", () => {
    const { container } = render(<Menu />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
