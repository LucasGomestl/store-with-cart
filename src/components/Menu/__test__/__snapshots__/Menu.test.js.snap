// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu tests Should render menu 1`] = `
<nav
  class="sc-AxjAm bngylr"
  data-testid="menu"
>
  <ul
    class="sc-AxirZ bWyXia"
  >
    <li>
      <a
        href="/"
      >
        Item 1
      </a>
    </li>
    <li>
      <a
        href="/"
      >
        Item 2
      </a>
    </li>
    <li>
      <a
        href="/"
      >
        Item 3
      </a>
    </li>
    <li>
      <a
        href="/"
      >
        Item 4
      </a>
    </li>
  </ul>
</nav>
`;
